pkg/kubelet/config/config.go

```diff
@@ -295,8 +300,9 @@ func (s *podStorage) merge(source string, change interface{}) (adds, updates, de
 	adds = &kubetypes.PodUpdate{Op: kubetypes.ADD, Pods: copyPods(addPods), Source: source}
 	updates = &kubetypes.PodUpdate{Op: kubetypes.UPDATE, Pods: copyPods(updatePods), Source: source}
 	deletes = &kubetypes.PodUpdate{Op: kubetypes.REMOVE, Pods: copyPods(deletePods), Source: source}
+	reconciles = &kubetypes.PodUpdate{Op: kubetypes.RECONCILE, Pods: copyPods(reconcilePods), Source: source}
 
-	return adds, updates, deletes
+	return adds, updates, deletes, reconciles
```

Comment:
(nit) 4 returns seems likes a lot. Consider either wrapping this in a `struct{adds, updates, deletes, reconciles}`, or returning a `map[kubetypes.PodOperation]kubetypes.PodUpdate`